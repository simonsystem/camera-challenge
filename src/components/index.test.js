import index from './';
import ReactDOM from 'react-dom';

jest.mock('react-dom');

test('default function calls ReactDOM.render', () => {
  index();
  expect(ReactDOM.render).toMatchSnapshot();
});