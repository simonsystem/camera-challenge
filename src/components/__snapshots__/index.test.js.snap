// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default function calls ReactDOM.render 1`] = `
[MockFunction] {
  "calls": Array [
    Array [
      <App />,
      null,
      [Function],
    ],
  ],
  "results": Array [
    Object {
      "type": "return",
      "value": undefined,
    },
  ],
}
`;
